capsules/src/rf233.rs

```diff
@@ -799,48 +891,85 @@ impl<'a, S: spi::SpiMasterDevice + 'a> radio::Radio for RF233<'a, S> {
     fn set_transmit_client(&self, client: &'static radio::TxClient) {
         self.tx_client.set(Some(client));
     }
+
     fn set_receive_client(&self, client: &'static radio::RxClient, buffer: &'static mut [u8]) {
         self.rx_client.set(Some(client));
         self.rx_buf.replace(buffer);
     }
 
+    fn set_config_client(&self, client: &'static radio::ConfigClient) {
```

Comment:
They can be. E.g., you want to send, but not receive packets. Or you just want to send packets, but don't want to configure the radio. A separate piece of code configures the radio.